pkg/virt-controller/watch/snapshot/source.go

```diff
@@ -183,23 +184,24 @@ func (s *vmSnapshotSource) Spec() (snapshotv1.SourceSpec, error) {
 		return snapshotv1.SourceSpec{}, err
 	}
 
-	var vmCpy *kubevirtv1.VirtualMachine
+	vmCpy := &snapshotv1.VirtualMachine{}
+	metaObj := metav1.ObjectMeta{
+		Name:            s.vm.Name,
+		Namespace:       s.vm.Namespace,
+		Annotations:     s.vm.Annotations,
+		Labels:          s.vm.Labels,
```

Comment:
looks like previous code copied labels/annotations into new maps.  Not sure it was really necessary.  But if we want to maintain that perhaps deepcopy `s.vm.ObjectMeta` then set managedFields to nil

Reply:
Done